pkg/gitlab/backend.go

```diff
@@ -0,0 +1,100 @@
+// Package dummy implements an example backend that can be used for testing
```

Comment:
Awesome mate! I think you could remove this comment tho, related to the `dummy` backend.